pkg/metric_storage/operation/operation.go

```diff
@@ -111,11 +115,11 @@ func ValidateMetricOperation(op MetricOperation) error {
 	var opErrs *multierror.Error
 
 	if op.Action == "" {
-		opErrs = multierror.Append(opErrs, fmt.Errorf("one of: 'action', 'set' or 'add' is required: %s", op))
+		opErrs = multierror.Append(opErrs, fmt.Errorf("one of: 'action', 'set', 'add' or 'observe' is required: %s", op))
```

Comment:
The original message is correct: the 'observe' shortcut is not implemented by this PR.